apps/web/src/pages/Profile/EmploymentEquityPage/components/EmploymentEquityForm/EmploymentEquityForm.tsx

```diff
@@ -57,7 +57,7 @@ const EmploymentEquityForm = ({
         },
         {
           label: intl.formatMessage(navigationMessages.stepOne),
-          url: paths.reviewApplication(applicationId ?? ""),
+          url: paths.application(applicationId ?? ""),
```

Comment:
The label for this breadcrumb should probably be updated, too.

Reply:
As discussed, the `applicationBreadcrumbs` are never actually rendered anymore since the application profile section forms are inline (see screenshot) and therefore no longer have breadcrumbs. Also, the profile section pages might be getting removed as part of https://github.com/GCTC-NTGC/gc-digital-talent/issues/6772, can you confirm @JamesHuf? If not, I can create a debt issue to remove them.

## Screenshot
![Screen Shot 2023-06-20 at 09 46 44](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/8fd2abce-1320-428b-bda9-ea558f8f57b6)

Reply:
Created issue for removal of profile section form page components: https://github.com/GCTC-NTGC/gc-digital-talent/issues/7019.